### PR TITLE
[bn] Localize partners/_index.html

### PR DESCRIPTION
**Pull request by itsPronay**
fixes issue : https://github.com/kubernetes/website/issues/44993
Related PR : https://github.com/kubernetes/website/pull/45009
Tracking Issue: https://github.com/kubernetes/website/issues/31677

### CODE REVIEW

File: content/bn/partners/_index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: পার্টনার
+bigheader: কুবারনেটিস পার্টনার
+abstract: কুবারনেটিস ইকোসিস্টেম বৃদ্ধি করা।
+class: gridPage
+cid: partners
+---
+
+<section id="users">
+    <h5>কুবারনেটিস পার্টনাররা একটি স্বস্তিকারী, জীবন্ত কোডবেস তৈরি করতে সহায়তা করে যা সমতুল্য প্ল্যাটফর্মের একটি স্পেক্ট্রাম সমর্থন করে।</h5>
```
Comment (asem-hamid) [inline content/bn/partners/_index.html:10]: i think শক্তিশালী is better suitable here

File: content/bn/partners/_index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          <button class="button landscape-trigger landscape-default" data-landscape-types="special--kubernetes-certified-service-provider" id="kcsp">KCSP পার্টনার দেখুন</button>
+          <br><br>আপনি যদি হতে চান
+          <a href="https://www.cncf.io/certification/kcsp/">KCSP</a>?
+        </center>
+      </div>
+      <div class="col-nav">
+        <center>
+          <h5>
+            <b>সার্টিফাইড কুবারনেটিস ডিস্ট্রিবিউশন, হোস্টেড প্ল্যাটফর্ম, এবং ইনস্টলার</b>
+          </h5>সফটওয়্যার সামঞ্জস্যতা নিশ্চিত করে যে প্রতিটি বিক্রেতার সংস্করণের কুবারনেটিস প্রয়োজনীয় API সমর্থন করে।
```
Comment (asem-hamid) [inline content/bn/partners/_index.html:28]: ভেন্ডরের

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Pull request merged**